src/parser/jobs/pld/modules/RoyalAuthority.tsx

```diff
@@ -0,0 +1,193 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {TieredRule, TARGET, Requirement} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import React from 'react'
+
+const SWORD_OATH_SEVERITY = {
+	89: TARGET.WARN,
+	94: TARGET.SUCCESS,
+}
+
+const DIVINE_MIGHT_SEVERITY = {
+	89: TARGET.WARN,
+	94: TARGET.SUCCESS,
+}
```

Comment:
please don't use `WARN`. i've been trying to get rid of it for years. i know, i know.

idea behind checklist is that it's either fine, or very much not fine. key rotational stuff that _must_ be done or you're dropping serious balls - i.e. not rolling your gcd. anything else should be a suggestion.

Reply:
I'm fine with getting rid of it, but is there something that can be said for that, so that it doesn't block commits like this?

Reply:
"is there something that can be said for that" What I mean by this is can this be a public notice and documented somewhere so people can start phasing it out of their modules? I don't feel like this should be a pr blocker because I'm not aware of any documentation that exists regarding phasing this out.

Reply:
It's... _sort_ of covered here: https://docs.xivanalysis.com/composing-job-analysis#checklist, but not explicitly. I should probably actually document it, but it's one of those things where the active developer base has more-or-less been around the entire time and nobody thought to do it, hah.